Lib/glyphs2ufo/torf_test.py

```diff
@@ -58,13 +58,6 @@ def test_style_nonregular_weight(self):
         self.assertEquals(
             build_style_name({'weight': 'Thin'}, 'weight', True), 'Thin Italic')
 
-    def test_postscript(self):
-        self.assertEquals(
-            build_postscript_name('MyFont', 'Regular'), 'MyFont-Regular')
-        self.assertEquals(
-            build_postscript_name('MyFont Condensed', 'Thin Italic'),
-            'MyFontCondensed-ThinItalic')
-
```

Comment:
Yea, this is intentional since `build_postscript_name` is being removed from torf.py along with other excess data-setting re: discussion at https://github.com/googlei18n/glyphs2ufo/issues/5